api/src/main/resources/liquibase-schema-only.xml

```diff
@@ -1831,7 +1831,7 @@
             <column name="email" type="varchar(255)">
             	<constraints unique="true"/>
             </column>
-            <column name="activation_key" type="varchar(50)"/>
```

Comment:
This changeset should be not be in the liquibase schema only file, should be in the 2.2 file